src/VisualStudio/Core/Def/Implementation/ProjectTelemetry/VisualStudioProjectTelemetryService.cs

```diff
@@ -82,12 +95,9 @@ private async Task StartAsync(CancellationToken cancellationToken)
             }
         }
 
-        private async Task StartWorkerAsync(CancellationToken cancellationToken)
+        private async Task StartWorkerAsync()
         {
-            _workQueue = new AsyncBatchingWorkQueue<ProjectTelemetryData>(
-                TimeSpan.FromSeconds(1),
-                NotifyTelemetryServiceAsync,
-                cancellationToken);
+            var cancellationToken = ThreadingContext.DisposalToken;
```

Comment:
Consider having StartWorkerAsync still take the cancellation token and then passing this in StartAsync(); I otherwise was scratching my head why StartAsync() was catching OperationCancelled exceptions when it wasn't passing a token in.